src/HandPose/index.js

```diff
@@ -58,6 +60,13 @@ class HandPose {
    * @private
    */
   async loadModel() {
+    this.modelName = handleModelName(
+      this.modelName,
+      ["MediaPipeHands"],
+      "MediaPipeHands",
+      "handPose"
+    );
+
```

Comment:
Personal preference but I feel like it's weird to set `this.modelName` to one value (the raw string) and then replace it with another value (the validated string).  I would move the `handleModelName` call into the constructor.  That way `this.modelName` is always the validated name and there is no confusion about what value it represents.

Reply:
Thank you @lindapaiste. This makes sense, I will update it.